.prettierrc

```diff
@@ -6,12 +6,6 @@
   "trailingComma": "es5",
   "useTabs": false,
   "overrides": [
-    {
```

Comment:
Why remove this? Is it inferred by prettier now?

Reply:
Yes. It's not necessary, TS is default for `.ts` and `.tsx`